packages/ghc/build.sh

```
@@ -0,0 +1,25 @@
+#!/bin/sh
+set -euo pipefail
```

Shellcheck (0.11.0)

[warning] In POSIX sh, set option pipefail is undefined.

(SC3040)